client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts

```diff
@@ -123,12 +123,7 @@ export const auditSettings: {
 }
 
 export const roundMocks: {
-  [key in
-    | 'empty'
-    | 'singleIncomplete'
-    | 'twoIncomplete'
-    | 'singleComplete'
-    | 'needAnother']: IRound[]
+  [key: string]: IRound[]
```

Comment:
I find the explicit defining of each state to be useful in IDE autocompletion, personally.

Reply:
Ohhh cool I was wondering why they were there. Thanks for the tip. I'll put them back. I wish the IDEs were smart enough to figure it out without the type annotation...